src/Chapter1.hs

```diff
@@ -652,8 +670,13 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+    | absN == 0 = 0
+    | absN < 10 = absN
```

Comment:
Zero cas is covered by `< 10` for the absolute value as well, so it is safe to remove this case:

```suggestion
    | absN < 10 = absN
```